Mathlib/Data/Bitvec/Basic.lean

```diff
@@ -92,34 +90,41 @@ theorem ofNat_toNat {n : ℕ} (v : Bitvec n) : Bitvec.ofNat _ v.toNat = v := by
     subst n
     simp only [Bitvec.ofNat, Vector.toList_cons, Vector.toList_nil, List.reverse_cons,
       Vector.toList_append, List.foldr]
-    erw [add_lsb_div_two, to_bool_add_lsb_mod_two]
+    erw [addLsb_div_two, decide_addLsb_mod_two]
     congr
-    apply ys_ih
+    rename_i tail_ih
+    apply tail_ih
     rfl
 #align bitvec.of_nat_to_nat Bitvec.ofNat_toNat
 
+set_option pp.coercions false
 theorem toFin_val {n : ℕ} (v : Bitvec n) : (toFin v : ℕ) = v.toNat := by
-  rw [to_fin, Fin.coe_of_nat_eq_mod, Nat.mod_eq_of_lt] <;> apply to_nat_lt
+  rw [toFin]
+  dsimp only [Fin.coe_ofNat_eq_mod]
```

Comment:
Does `Fin.coe_ofNat_eq_mod` not work in the `rw` any more?

Reply:
Yeah it worked now. Changed it